pkg/updatechecker/updatechecker.go

```diff
@@ -235,6 +235,8 @@ func CheckForUpdates(opts CheckForUpdatesOpts) (*UpdateCheckResponse, error) {
 		ReportingInfo:            reporting.GetReportingInfo(a.ID),
 	}
 
+	thisUpdateCheckAt := time.Now()
```

Comment:
nit pick
```suggestion
	lastUpdateCheckAt := time.Now()
```

Reply:
that variable name is already taken, and used for the last time we checked for updates successfully

Reply:
ah